core-java-modules/core-java-arrays-3/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
According to eclipse formatter, the indentation is also off here. Thanks.

Reply:
Done